### ISSUE TITLE
NullPointerException on launch

### ISSUE THREAD

**Issue by Venryx** [nextcloud/android#8431]
Error happened on opening the app, after a while of not using it.

************ CAUSE OF ERROR ************

java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter password
	at okhttp3.Credentials.basic(Unknown Source:8)
	at okhttp3.Credentials.basic$default(Credentials.kt:28)
	at okhttp3.Credentials.basic(Unknown Source:2)
	at com.owncloud.android.lib.common.OwnCloudClientFactory.createNextcloudClient(OwnCloudClientFactory.java:217)
	at com.owncloud.android.operations.RefreshFolderOperation.updatePredefinedStatus(RefreshFolderOperation.java:338)
	at com.owncloud.android.operations.RefreshFolderOperation.updateCapabilities(RefreshFolderOperation.java:311)
	at com.owncloud.android.operations.RefreshFolderOperation.updateOCVersion(RefreshFolderOperation.java:277)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:230)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:359)
	at java.lang.Thread.run(Thread.java:764)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30160090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Xiaomi
Device: perseus
Model: Mi MIX 3
Id: PKQ1.180729.001
Product: perseus

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: V10.3.2.0.PEEMIXM

**Comment by Venryx** [comment]
Further info: The crash only happens when I have the "nextcloud05.webo.cloud" account selected. If I quickly change the account (when the app is loading) to my "us.cloudamo.com" account, the error does not occur.

When I then visited `nextcloud05.webo.cloud` manually, this message showed: "We have detected multiple invalid login attempts from your IP. Therefore your next login is throttled up to 30 seconds."

Thus, my guess is that my free account at `webo.cloud` was either disabled, or hit some other limitation, causing either the sign-in to be rejected, or the sign-in process to error out part-way. (I can't confirm if my account is still active atm because manual sign-in also keeps being rejected, with the message: "Too many requests. There were too many requests from your network. Retry later or contact your administrator if this is an error.")

That said, it would be good of the Nextcloud system can gracefully handle rejected/errored sign in attempts (and present a useful message), rather than the current error popup.
